### ISSUE TITLE
Nullable Microsoft.OData.Edm.Date

### ISSUE THREAD

**Issue by LarsBauer** [moh-hassan/odata2poco#32]
Hi,

is there are specific reason why nullable `Microsoft.OData.Edm.Date` is not implemented?
If I understand your source correctly this should just be a small change in `OData2PocoLib/Helper.cs`
```csharp
        /// <summary>
        /// Nullable DataType
        /// </summary>
        public static readonly Dictionary<string, string> NullableDataTypes = new Dictionary<string, string>
      {
            {"object" , ""},
            {"string" , ""},
            {"bool" , "?"},
            {"byte" , "?"},
            {"char" , "?"},
            {"decimal" , "?"},
            {"double" , "?"},
            {"short" , "?"},
            {"int" , "?"},
            {"long" , "?"},
            {"sbyte" , "?"},
            {"float" , "?"},
            {"ushort" , "?"},
            {"uint" , "?"},
            {"ulong" , "?"},
            {"void" , "?"},
            ////Nullable DateTime issue #3 , included in v2.3.0
            {"DateTime","?" },
            {"DateTimeOffset","?" },
            {"TimeSpan","?" },
            {"Guid","?" },
            // THIS IS NEW
            {"Microsoft.OData.Edm.Date","?"},
            {"Microsoft.OData.Edm.TimeOfDay","?"}
        };
```

Can I provide a pull request for this feature?
Thanks in advance!

**Comment by moh-hassan** [comment]
Thanks for suggestion.
You can provide PR and nice to add also Microsoft.OData.Edm.DateTimeOffset

**Comment by LarsBauer** [comment]
Thanks for the quick response.
I just opened the PR. The type `Microsoft.OData.Edm.DateTimeOffset` does not exist. I think it gets resolved to regular `DateTimeOffset`.

**Comment by moh-hassan** [comment]
Ok, thanks

**Comment by LarsBauer** [comment]
Hey, any plans to merge my PR?

**Comment by moh-hassan** [comment]
Hi @LarsBauer 
PR is merged and a new package v3.4.2 is available.
